tests/test_ember_osf_web.py

```diff
@@ -29,28 +29,23 @@ def test_dont_use_ember_app(self, mock_use_ember_app):
 
         assert not mock_use_ember_app.called
 
-    @mock.patch('website.ember_osf_web.decorators._get_current_user')
-    @mock.patch('website.ember_osf_web.decorators.waffle.flag_is_active', return_value=True)
+    @mock.patch('api.waffle.utils._get_current_user')
     @mock.patch('website.ember_osf_web.decorators.use_ember_app')
-    def test_ember_flag_is_active_authenticated_user(self, mock_use_ember_app, mock_flag_is_active, mock__get_current_user):
+    def test_ember_flag_is_active_authenticated_user(self, mock_use_ember_app, mock__get_current_user):
         user = UserFactory()
         mock__get_current_user.return_value = user
 
         ember_flag_is_active('active_flag')(self.mock_func)()
 
-        mock_flag_is_active.assert_called_with(request, 'active_flag')
         assert request.user == user
         mock_use_ember_app.assert_called_with()
 
-    @mock.patch('website.ember_osf_web.decorators._get_current_user', return_value=None)
-    @mock.patch('website.ember_osf_web.decorators.waffle.flag_is_active', return_value=True)
     @mock.patch('website.ember_osf_web.decorators.use_ember_app')
-    def test_ember_flag_is_active_unauthenticated_user(self, mock_use_ember_app, mock_flag_is_active, mock__get_current_user):
+    def test_ember_flag_is_active_unauthenticated_user(self, mock_use_ember_app):
         ember_flag_is_active('active_flag')(self.mock_func)()
         group = Group.objects.create(name='foo')
 
         self.flag.groups.add(group)
 
-        mock_flag_is_active.assert_called_with(request, 'active_flag')
```

Comment:
Instead of deleting this portion of the test, can you rework so we're mocking your new `flag_is_active`?

Reply:
👍

Reply:
@Johnetordoff, I'm not seeing where this test was restored,